### PR TITLE
Allow setting solr host ip

### PR DESCRIPTION
**Pull request by gadelkareem**
Useful incase you want to restrict access by IP in the jetty engine settings`<Set name="host"><Property name="jetty.host" default="{{solr_host}}" /></Set>`

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ solr_service_name: solr
 solr_install_dir: /opt
 solr_install_path: "/opt/{{ solr_service_name }}"
 solr_home: /var/solr
+solr_ip: "127.0.0.1"
```
Comment (geerlingguy) [inline defaults/main.yml]: Can you change the variable name to `solr_host` instead, and default it to `localhost`? That way it maintains backwards compatibility, and can also just be a hostname like `solr.example.com`.

Comment (gadelkareem) [inline defaults/main.yml]: Done!

**Comment by geerlingguy** [comment]
@gadelkareem wow, that was quick! Will merge once tests pass.

**Comment by geerlingguy** [comment]
@gadelkareem - Note that I've changed the variable name yet again to `solr_connect_host`, because I forgot we already use `solr_host` for Solr 4 and below. Oops! Fixed in this commit: https://github.com/geerlingguy/ansible-role-solr/commit/f6ed5f8a1c3237bac388f3744168bac7441c3b2d

**Pull request merged**